R/appendMeanMatF.R

```diff
@@ -10,7 +10,8 @@
 #' YearPublication, DOI.ISBN, SpeciesAuthor, MatrixPopulation, and
 #' MatrixDimension.
 #' 
-#' @param db A COM(P)ADRE database object.
+#' @param db A COM(P)ADRE database object. Database versions <=4.0.1 will be coerced
```

Comment:
Will this apply to both databases? compadre is at 4.0.1 but comadre is only at 2.0.1

Reply:
You're right @iainmstott. This needs to apply to both DBs, so "COMADRE versions <=4.0.1 and COMPADRE versions <=2.0.1 will be coerced"